### ISSUE TITLE
Next song queue scroll reset when >1 songs displayed

### ISSUE THREAD

**Issue by faustaahmad** [fast4x/RiMusic#3080]
### Steps to reproduce the bug

- `settings` Enable Show next song queue
- `settings` Set Songs number to show more than 1
- `player` play a song, then try scroll on next song queue

### Expected behavior

Scroll just fine like when using Set Songs number to show = 1

### Actual behavior

Scroll position reset to initial position when Set Songs number to show > 1

### RiMusic version

0.6.46

### Android version

Android 11

**Comment by twistios** [comment]
I am not sure if I understand the report correctly.
I observe something similar, but this might be intended behaviour (kind of at least). For me it jumps to the start when I move the finger too much to the left, there is the "return to correct position" button and I think that it gets activated then.

Something that might be a good idea (if it like it seems to me) would be to not activate this button if currently scrolling the mini queue.

If it is someting else @faustaahmad, please tell.

**Comment by faustaahmad** [comment]
I couldnt explain it with words

https://github.com/user-attachments/assets/00522a19-f94d-4a1b-953c-bc56880f72cf

**Comment by twistios** [comment]
When words are not enough...
I guess this is the reason why the "Screenshots/Screen recordings" section exists. :smile: 

It looks to me that you are not scrolling enough (i.e. you have to start swiping more on the right). If this does not work, then this probably nexts some fix. 
Or it could be that you are swiping too much to the left (where the already mentioned button is located).

You can say if this fixes it for you or @fast4x has to look at this to say if it is intended or a problem.

**Comment by aneesh1122** [comment]
> you are not scrolling enough

**Comment by faustaahmad** [comment]
> you are not scrolling enough

Dont expect me to scroll from one edge to other edge's end

**Comment by aneesh1122** [comment]
> > you are not scrolling enough
> 
> Dont expect me to scroll from one edge to other edge's end

Don't expect me to solve your problems ✌️

**Comment by aneesh1122** [comment]
> > you are not scrolling enough
> 
> Dont expect me to scroll from one edge to other edge's end

![spiderman-missed.gif](https://github.com/user-attachments/assets/70440a7d-e944-4955-9c97-cc4e5a0dd531)

**Comment by faustaahmad** [comment]
Here with visual taps, is it hard enough?
https://gofile.io/d/RYiGQp
Github hit filesize limit

**Comment by fast4x** [comment]
We will try to improve it over time, for the moment it is not a priority.

**Comment by aneesh1122** [comment]
> Here with visual taps, is it hard enough? https://gofile.io/d/RYiGQp Github hit filesize limit

Alright I've fixed this bug however if you act rude towards me again I won't fix anything for you. Peace

**Comment by faustaahmad** [comment]
Im sorry if my words was rude
I dont know how much scrolling are enough, I have tried the suggested solution above but the list keep coming back to the previous position

Thank you

**Comment by aneesh1122** [comment]
> Im sorry if my words was rude
> I dont know how much scrolling are enough, I have tried the suggested solution above but the list keep coming back to the previous position
> 
> Thank you

I've fixed it. Now the list will come back only after the song is changed.

https://github.com/user-attachments/assets/d930b194-2bb1-49ec-8545-b5e9cf3c0afe

**Comment by fast4x** [comment]
Available from version 0.6.47
